File: src/main/java/com/maranin/kotlinfundemo/KotlinFunDemoApplication.java
package com.maranin.kotlinfundemo;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@SpringBootApplication
@EnableJpaRepositories
@EntityScan
public class KotlinFunDemoApplication {

	public static void main(String[] args) {
		SpringApplication.run(KotlinFunDemoApplication.class, args);
	}

}

File: src/main/kotlin/com/maranin/kotlinfundemo/KotlinFunDemoApplication.kt
package com.maranin.kotlinfundemo

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.data.jpa.repository.config.EnableJpaRepositories
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableJpaRepositories
@EntityScan
class KotlinFunDemoApplication

fun main(args: Array<String>) {
    runApplication<KotlinFunDemoApplication>(*args)
}
